src/components/common/index/MainCountdown.tsx

```diff
@@ -30,20 +31,24 @@ export const MainCountdown: React.FC<MainCountdownProps> = (props) => {
     };
   }, [departureTime]);
 
+  const isPast = (): boolean => {
+    return departureTime < now;
+  };
```

Comment:
関数にした理由ってある？？
`const isPast = departureTime < now;`

Reply:
@yuzuki-aritomo 
pushしようとするとなぜかrejectされるんやけどなんでかわかる？
```
To https://github.com/yuzuki-aritomo/train_front
 ! [rejected]        HEAD -> yamashita/#39/change-countdown-color (non-fast-forward)
error: failed to push some refs to 'https://github.com/yuzuki-aritomo/train_front'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Reply:
あ、できたわ